cluster/options.go

```diff
@@ -9,19 +11,22 @@ import (
 type DriverOpts []string
 
 // String returns a string from the driver options
-func (do DriverOpts) String(key string) (string, bool) {
+func (do DriverOpts) String(key, env string) (string, bool) {
 	for _, opt := range do {
 		kv := strings.SplitN(opt, "=", 2)
 		if kv[0] == key {
 			return kv[1], true
 		}
 	}
+	if env := os.Getenv(env); env != "" {
```

Comment:
Uhm, you are right. We could replace `.` with `_` so it'd be `SWARM_CLUSTER_OPT_SWARM_OVERCOMMIT` but maybe that's overkill?